test/integration/action_execution.go

```diff
@@ -90,6 +91,9 @@ var _ = Describe("Action Executor ", func() {
 
 	Describe("executing action move pod", func() {
 		It("should result in new pod on target node", func() {
+			if framework.TestContext.IsOpenShiftTest {
+				Skip("Ignoring pod move case for the deployment.")
```

Comment:
s/Ignoring pod move case for the deployment/Ignoring pod move case for deployment against openshift target
Also please change all the other ignore messages on similar lines

Reply:
done